package.json

```diff
@@ -13,7 +13,11 @@
     "start": "ng serve",
     "build": "ng build --prod --base-href '/elements-starter-angular'",
     "postinstall": "ngcc --tsconfig './tsconfig.app.json'",
-    "lint": "tslint 'src/**/*.ts'"
+    "lint": "tslint 'src/**/*.ts'",
+    "serve-prod": "angular-http-server --path ./dist/angular -p 4200",
+    "e2e:execute": "cypress run",
+    "e2e:open": "cypress open",
+    "e2e:run": "start-server-and-test serve-prod http://localhost:4200 e2e:execute"
```

Comment:
What's the benefit of using `start-server-and-test`? Shouldn't it run just with `serve-prod`?

Reply:
`start-server-and-test` is a simple script that orchestrates the following:
1. Starts `yarn serve-prod`
2. Waits until it is up and running by pinging `http://localhost:4200`
3. When it is, it executes `yarn e2e:execute`
4. When `e2e:execute` quits, it tears down the server.